test/libsolidity/smtCheckerTests/types/struct/struct_recursive_3.sol

```diff
@@ -28,11 +28,11 @@ contract C {
 	}
 }
 // ----
-// Warning 6328: (124-144): Assertion violation happens here.
-// Warning 6328: (148-182): Assertion violation happens here.
-// Warning 6328: (186-216): Assertion violation happens here.
-// Warning 6328: (220-264): Assertion violation happens here.
-// Warning 6328: (268-308): Assertion violation happens here.
+// Warning 6328: (124-144): CHC: Assertion violation happens here.
+// Warning 6328: (148-182): CHC: Assertion violation happens here.
+// Warning 6328: (186-216): CHC: Assertion violation happens here.
+// Warning 6328: (220-264): CHC: Assertion violation happens here.
+// Warning 6328: (268-308): CHC: Assertion violation happens here.
 // Warning 8115: (81-85): Assertion checker does not yet support the type of this variable.
```

Comment:
@leonardoalt Where is this emitted?

Reply:
SMTEncoder which is shares by BMC and CHC. That's why I didn't add prefixes there